Spec0/readme.md

```diff
@@ -0,0 +1,8 @@
+# Emojisum
+
+This is the Spec0 emojisum deterministic checksums. It is shared by all.
+It assumes a 256bit input, for example SHA256 or Blake2b at 32bytes.
```

Comment:
There is no point in pinning any specific hashing, as we only operate on the hex encoded form.

Reply:
oki I agree